aruco_detect/cfg/DetectorParams.cfg

```diff
@@ -71,7 +71,7 @@ gen.add("minMarkerPerimeterRate",                 double_t, 0,
 
 gen.add("maxMarkerPerimeterRate",                 double_t, 0,
         "Determine maximum perimeter for marker contour to be detected. This is defined as a rate respect to the maximum dimension of the input image",
-        4.0, 0, 1)
+        4.0, 0, 4.0)
```

Comment:
I would maybe consider changing the max value from `4.0` to `4` since all other max values here are integers. Or even change it to `10`.